com.unity.mobile.notifications/Editor/AndroidNotificationPostProcessor.cs

```diff
@@ -119,6 +119,7 @@ internal static void InjectReceivers(string manifestPath, XmlDocument manifestXm
             {
                 notificationRestartOnBootReceiver = manifestXmlDoc.CreateElement("receiver");
                 notificationRestartOnBootReceiver.SetAttribute("name", kAndroidNamespaceURI, kNotificationRestartOnBootName);
+                notificationRestartOnBootReceiver.SetAttribute("exported", kAndroidNamespaceURI, "false");
```

Comment:
Why is this necessary?
This same attribute should be set a bit below, on line 133, except that it probably needs to be changes to false.